commons/constants.py

```diff
@@ -468,16 +468,23 @@ class Sizes:
 
 #errors for rgw and cortx s3
 RGW_ERR_LONG_OBJ_NAME = ("An error occurred (InvalidObjectName) when calling the PutObject "
-"operation: Unknown")
+                         "operation: Unknown")
 CORTX_ERR_LONG_OBJ_NAME = ("An error occurred (KeyTooLongError) when calling the PutObject "
-"operation: Your key is too long.")
+                           "operation: Your key is too long.")
 RGW_ERR_WRONG_JSON = ("An error occurred (InvalidPart) when calling the CompleteMultipartUpload "
-"operation: Unknown")
+                      "operation: Unknown")
 CORTX_ERR_WRONG_JSON = ("An error occurred (InvalidPart) when calling the CompleteMultipartUpload "
-"operation: One or more of the specified parts could not be found. The part might not have been "
-"uploaded, or the specified entity tag might not have matched the part's entity tag.")
+                        "operation: One or more of the specified parts could not be found. "
+                        "The part might not have been uploaded, or the specified entity tag"
+                        " might not have matched the part's entity tag.")
 RGW_ERR_DUPLICATE_BKT_NAME = ("An error occurred (BucketAlreadyExists) when calling the "
-"CreateBucket operation: Unknown")
+                              "CreateBucket operation: Unknown")
 CORTX_ERR_DUPLICATE_BKT_NAME = ("An error occurred (BucketAlreadyOwnedByYou) when calling"
-"the CreateBucket operation: Your previous request to create the named bucket succeeded and "
-"you already own it.")
+                                "the CreateBucket operation: Your previous request to create the "
+                                "named bucket succeeded and you already own it.")
+RGW_ERR_NO_SUCH_UPLOAD = "An error occurred (NoSuchUpload) when calling the UploadPart operation: "\
```

Comment:
Let's make this consistent with the line wrapping approach used for the other strings.